packages/astro/test/units/assets/fonts/load.test.js

```diff
@@ -82,4 +90,10 @@ it('loadFonts()', async () => {
 	assert.equal(Array.from(hashToUrlMap.keys()).length > 0, true);
 	assert.deepStrictEqual(Array.from(resolvedMap.keys()), ['Roboto']);
 	assert.deepStrictEqual(logs, ['Fonts initialized']);
+	assert.equal(
```

Comment:
nit: if you manage to use `assert.match`, use that because it shows the string to evaluate in case there's an error

Reply:
I'm going to look into this thanks!